### ISSUE TITLE
2x2 matrix inverse example/remark clarity

### ISSUE THREAD

**Issue by vatai** [mml-book/mml-book.github.io#504]
**Is your feature request related to a problem? Please describe.**
During our reading sessions with Machine Learning Tokyo, some people (some of them independently on multiple occasions) got confused about the Remark after the definition of the matrix inverse: **Page 24, Section 2.2.2**, (2.22) `B=[...]`, 2020-03-28 version of the pdf. I understand the whys and hows of the remark, but for novices it might be confusing, because it is not entirely a concrete example (with concrete numbers) and and also it not an entirely abstract example (it basically uses the `inv(A) = det(A)^{-1} adj(A)` identity for 2x2 matrices without explicitly stating it).
 
**Describe the solution you'd like**
The least invasive thing would be to change the variable name in (2.22) from `B` to something else (suggestions `M`, `A'`). Explanation: some readers were confused if the `B` from (2.22) is the same `B` as the `B` in the definition just above (2.22).

Also I'd call it an Example instead of a Remark, and add a note/remark saying that this is the special case of the `inv(A) = det(A)^{-1} adj(A)` identity).

**Describe alternatives you've considered**
A more invasive approach would be to add a concrete examples of an `A` matrices which has an inverse. Show (2.23) (i.e. `A . adj(A) = det(A) . I`) and the calculation with concrete numbers which demonstrate the `A . A^{-1} = I`. Then introduce the abstract 2x2 case currently described in the book, and (maybe) add another concrete example `A` matrix which doesn't have an inverse.

**Comment by mayankbhaskar007** [comment]
^ This is a problem, that me and some members of my team had difficulties in processing.
Basically, if the author can give some clarity behind the intuition as recommended by @vatai, it will be great for folks who come in from different backgrounds and/or other ways, the same concept was taught to them

**Comment by mpd37** [comment]
Good point regarding the overloading of notation. Agree that "B" is an unfortunate naming convention in the remark/example.
I'll change B into A' as you suggested.

We wanted to leave this as a remark because of the last sentence, which states when the inverse of a 2x2 matrix exists.

I will leave the concrete example as a feature request for the time when we can make more significant changes to the book. 

Thanks a lot for your suggestion!
